script.js

```diff
@@ -264,13 +288,13 @@ function sendDestinationDetails() {
 function updateDestination(desID) {
     var desID = desID;
```

Comment:
This variable assignment operation is useless. You can directly use desID local variables.